data/json/items/comestibles/mutagen.json

```
@@ -3,7 +3,7 @@
     "abstract": "mutagen_flavor",
     "type": "COMESTIBLE",
     "name": { "str": "abstract mutagen flavor" },
-    "description": "A rare substance of uncertain origins.  Causes you to mutate.",
+    "description": "A slurry of organic chemicals and compounds, in half-processed form.  In high enough dose, it will cause you to mutate, but it would be safer and more predictable if further refined into a primer.",
```

OK so I've been focused in other places  but the sentence below implies that if possible the player should only use primers to mutate.  Is that currently true?   Last I checked you needed both the primer and the base mutagen vitamins to mutate or you needed just base mutagen vitamins?   It's cool if this has changed but I want to make sure of it before we make statements like the one below.
```suggestion
    "description": "A slurry of organic chemicals and compounds, in half-processed form.  In high enough dose, it will cause you to mutate, but it would be safer and more predictable if further refined into a primer.",
```

---

You are correct that you can't use primers only to mutate, but this is the description for slurry, not for primer. Slurry (eventually) provides both catalytic factor and typed factor so it will cause mutations on its own, although it's inefficient and unhealthy to do that (it also increases toxins and instability).